File: sdk/javascript/scripts/sbm-common/init-param-animation.js
var initParamAnimation = {
	initParamAnimation : function() {
		// locomotion main state
		stateAllLocomotion.locomotionSetup("test_utah.sk", "base", "", "all")

		// starting state, starting locomotion with different angle
		stateAllStarting.startingSetup("test_utah.sk", "base", "", "all")

		// idle turn state, facing adjusting
		stateAllIdleTurn.idleTurnSetup("test_utah.sk", "base", "", "all")

		// step state, stepping adjusting
		stateAllStep.stepSetup("test_utah.sk", "base", "", "all")

		// transitions
		transitions.transitionSetup("", "all")
		//scene.run("stateSimple.py")
	}
}

File: data/sbm-common/scripts/init-param-animation.py
# locomotion main state
scene.run("stateAllLocomotion.py")
locomotionSetup("test_utah.sk", "base", "", "all")

# starting state, starting locomotion with different angle
scene.run("stateAllStarting.py")
startingSetup("test_utah.sk", "base", "", "all")

# idle turn state, facing adjusting
scene.run("stateAllIdleTurn.py")
idleTurnSetup("test_utah.sk", "base", "", "all")

# step state, stepping adjusting
scene.run("stateAllStep.py")
stepSetup("test_utah.sk", "base", "", "all")

# transitions
scene.run("transitions.py")
transitionSetup("", "all")
# scene.run("stateSimple.py")
